index.js

```diff
@@ -277,19 +32,38 @@ function words(options) {
     return wordList[randInt(wordList.length)];
   }
 
+  // limits the size of words to the minimum and maximum possible
+  function limitWordSize(wordSize) {
+    if (wordSize < shortestWordSize) wordSize = shortestWordSize;
+    if (wordSize > longestWordSize) wordSize = longestWordSize;
+    return wordSize;
+  }
+
   // random int as seeded by options.seed if applicable, or Math.random() otherwise
   function randInt(lessThan) {
     const r = random ? random() : Math.random();
     return Math.floor(r * lessThan);
   }
 
   // No arguments = generate one word
-  if (typeof(options) === 'undefined') {
+  if (typeof options === "undefined") {
     return word();
   }
 
+  // Generate one word with limits
+  if (
```

Comment:
In this code block, only the first if statement is hit. The two blocks are essentially the same. In fact, in the current tests, neither of these blocks get hit because there are always additional options being passed. Can you eliminate the first code block, please.